### ISSUE TITLE
Incorrect type

### ISSUE THREAD

**Issue by gabrielozaki** [braintree/credit-card-type#107]
### General information

- Library version: 8.3.0
- Browser and OS: Running on a Ubuntu  19.10 with Nodejs 12.16.1 

### Issue description
I have this test bin: "504175" and it's a ELO, confirmed by ccbins, freebinchecker, bincodes and the legacy system that i was working.
But I'm getting a Maestro result:
```JS
const creditCardType = require('credit-card-type');

const eloCard = creditCardType('504175');
console.log(eloCard);
```
Output:
```JS
[ { niceType: 'Maestro',
    type: 'maestro',
    patterns: [ 493698, [Array], [Array], [Array], 63, 67, 6 ],
    gaps: [ 4, 8, 12 ],
    lengths: [ 12, 13, 14, 15, 16, 17, 18, 19 ],
    code: { name: 'CVC', size: 3 },
    matchStrength: 6 } ]
```

There any place where i can get a updated bin table and make a PR to correct this issue?

**Comment by crookedneighbor** [comment]
Was able to reproduce this.

`504175` is listed as an elo card number: https://github.com/braintree/credit-card-type/blob/401f82161e630beec72903370313898230e25cbc/lib/card-types.js#L163

So it's likely that the maestro ranges need to be updated to accommodate it. Right now, maestro is showing a an identical match strength for `504175` with ELO, and since it has a higher preference in the card test order, it's getting displayed.

Should be as simple as changing this line: https://github.com/braintree/credit-card-type/blob/401f82161e630beec72903370313898230e25cbc/lib/card-types.js#L137

Maestro will need to split that line into 2 ranges:

```js
[500000, 504174],
// 504175 is confirmed to be only ELO cards
[504176, 506698],
```

Note: we're in the process of [migrating this lib to Typescript](https://github.com/braintree/credit-card-type/pull/105), so we're unlikely to merge in any changes until that PR is complete.

**Comment by gabrielozaki** [comment]
Hi, 
I will close the issue util the Typescript migration is complete
Thank you for your time and attention :smile: 
Gabriel Ozaki

**Comment by crookedneighbor** [comment]
I'm going to leave this open so we can better track it.

**Comment by crookedneighbor** [comment]
Typescript migration is complete.

**Comment by crookedneighbor** [comment]
This was released as part of v9.0.1
